install_config/install/docker_registry.adoc

```diff
@@ -173,20 +173,52 @@ $ oc logs docker-registry-1-da73t
 
 == File Storage
 
-Tag and image metadata is stored in OpenShift, but the registry owns layer and
-signature data. The registry stores data in a volume that is mounted into the
-registry container at *_/registry_*. You can
+Tag and image metadata is stored in OpenShift, but the registry stores layer and
+signature data in a volume that is mounted into the registry container at
+*_/registry_*.
+
+To see a registry's contents, ssh into the pod, then list the contents:
+
+. List the current pods to find the desired pod name:
++
+----
+# oc get pods
+----
++
+You can also describe the node to find the node hostname used for ssh:
++
+----
+# oc describe pod <node_name>
+----
+
+. Log into the desired node:
++
+----
+# ssh node.example.com
+----
+
+. List all the Docker containers on the node host:
++
+----
+# docker ps
+----
+
+. List the registry contents using the `docker exec` command:
++
+----
+# docker exec -it <docker_id> find /registry
```

Comment:
Wasn't the original point of this BZ that `find` is not in the latest registry container? Is it now? (Haven't checked yet)